app/classifier/tasks/single.jsx

```diff
@@ -0,0 +1,165 @@
+import { Markdown } from 'markdownz';
+import React from 'react';
+import GenericTask from './generic';
+import GenericTaskEditor from './generic-editor';
+
+const NOOP = Function.prototype;
+
+class Summary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.expand = this.expand.bind(this);
+    this.collapse = this.collapse.bind(this);
+    this.state = {
+      expanded: this.props.expanded
+    };
+  }
+
+  expand() {
+    this.setState({ expanded: true });
+  }
+
+  collapse() {
+    this.setState({ expanded: false });
+  }
+
+  render() {
+    let toggleButton = <button type="button" className="toggle-more" onClick={this.expand}>More</button>;
+    let answers = <div className="answer">No answer</div>;
+    if (!this.state.expanded && this.props.annotation.value) {
+      answers = (
+        <div className="answer">
+          <i className="fa fa-check-circle-o fa-fw"></i>
+          <Markdown tag="span" inline={true}>{this.props.task.answers[this.props.annotation.value].label}</Markdown>
+        </div>
+      );
+    }
+    if (this.state.expanded) {
+      toggleButton = <button type="button" className="toggle-more" onClick={this.collapse}>Less</button>;
+      answers = []
+      for (const [i, answer] of this.props.task.answers.entries()) {
+        if (!answer._key) {
+          answer._key = Math.random();
+        }
+        let icon = <i className="fa fa-circle-o fa-fw"></i>;
+        if (i === this.props.annotation.value) {
+          icon = <i className="fa fa-check-circle-o fa-fw"></i>;
+        }
+        answers.push(
+          <div key={answer._key} className="answer">
+            {icon}
+            <Markdown tag="span" inline={true}>{answer.label}</Markdown>
+          </div>
+        );
+      }
+    }
+    return (
+      <div>
+        <div className="question">
+          <Markdown>
+            {this.props.task.question}
+          </Markdown>
+          {toggleButton}
+        </div>
+        <div className="answers">
+          {answers}
+        </div>
+      </div>
+    );
+  }
+}
+
+Summary.propTypes = {
+  task: React.PropTypes.object,
+  annotation: React.PropTypes.object,
+  expanded: React.PropTypes.bool
+};
+
+Summary.defaultProps = {
+  task: null,
+  annotation: null,
```

Comment:
Oh, good call.  As for the task components, the `getDefaultAnnotation` method is called via the `preparetoclassify` method on mount (https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/classifier/index.cjsx#L76, https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/classifier/index.cjsx#L143, https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/classifier/index.cjsx#L499), so the `defaultProp` will never be used.  By extension the `Summary` component should always have a non-null annotation object.

It might be in order to remove the `Summary`'s default annotation and just make the prop `isRequired`.